src/client/pages/Checkout/CheckoutPayment/ContactInformation/ContactInformation.tsx

```diff
@@ -49,7 +47,19 @@ const Spacer = styled.div`
   height: 1.5rem;
 `
 
-export const ContactInformation = (data: ContactInfoData) => {
+const HorizontalDivider = styled(Divider)`
+  margin: 1.5rem 0;
+  ${MEDIUM_SMALL_SCREEN_MEDIA_QUERY} {
+    margin: 3rem 0;
+  }
+`
+
+export const ContactInformation = ({
+  firstName,
```

Comment:
You could default here to empty string if you want I believe. Same with last name
```suggestion
  firstName = '',
```